waveshare-epd/epd2in13/epd2in13.go

```diff
@@ -274,5 +298,28 @@ func (d *Device) ClearBuffer() {
 
 // Size returns the current size of the display.
 func (d *Device) Size() (w, h int16) {
+	if d.rotation == 1 || d.rotation == 3 {
+		return d.height, d.width
+	}
 	return d.width, d.height
 }
+
+// SetRotation changes the rotation of the device
```

Comment:
I think it would be useful to add what the rotation number means. For example: "pass rotation as a number 0-3 that is interpreted as 0, 90, 180, and 270 degrees respectively".